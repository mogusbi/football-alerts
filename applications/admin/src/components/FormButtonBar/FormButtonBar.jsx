import PropTypes from 'prop-types';
import React, {memo} from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
  margin-top: 24px;
`;

function FormButtonBar ({children}) {
  return (
    <Wrapper>
      {children}
    </Wrapper>
  );
}

FormButtonBar.propTypes = {
  children: PropTypes.node.isRequired
};

export default memo(FormButtonBar);
