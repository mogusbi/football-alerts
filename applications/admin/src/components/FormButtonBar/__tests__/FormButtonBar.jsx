import {mount} from 'enzyme';
import React from 'react';
import FormButtonBar from '../FormButtonBar';

describe('FormButtonBar', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(
      <FormButtonBar>
        <button type='button'>
          Test button
        </button>
      </FormButtonBar>
    );
  });

  it('should render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
