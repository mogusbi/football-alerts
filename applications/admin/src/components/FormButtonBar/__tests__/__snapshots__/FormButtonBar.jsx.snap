// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormButtonBar should render component 1`] = `
.c0 {
  margin-top: 24px;
}

<FormButtonBar>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  margin-top: 24px;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="c0"
      >
        <button
          type="button"
        >
          Test button
        </button>
      </div>
    </StyledComponent>
  </styled.div>
</FormButtonBar>
`;
